src/runmode-napatech.c

```diff
@@ -200,16 +190,29 @@ static int NapatechInit(int runmode)
         exit(EXIT_FAILURE);
     }
 
-    switch(runmode) {
+    struct NapatechStreamDevConf *conf = SCMalloc(sizeof (struct NapatechStreamDevConf));
```

Comment:
I prefer SCCalloc as that initializes the memory to 0. Reduces chance of issues.